src/GasExposureAnalytics.py

```diff
@@ -35,6 +35,14 @@
 SAFE_ROUNDING_FACTOR_PROPERTY = 'safe_rounding_factor'
 GAS_LIMITS_PROPERTY = 'gas_limits'
 
+# Sensor range limitations. These are intentionally hard-coded and not configured. They're used to cross-check
+# that the PPM limits configured for each time-window respects the sensitivity range of the sensors.
+SENSOR_RANGE_PPM  = {
+    'carbon_monoxide'  : {'min' : 1   , 'max' : 1000}, # CJMCU-4541 / MICS-4514 Sensor
+    'nitrogen_dioxide' : {'min' : 0.05, 'max' : 10  }  # CJMCU-4541 / MICS-4514 Sensor
```

Comment:
Will these limits be used anywhere else? If so, might be a good idea to store in a json file even if they are hardcoded and not leave in the code here. It can be a different file than the prometeo_config.json. Maybe `prometeo_hard_limits.json`? @JSegrave-IBM

Reply:
Yup, sounds good. Since it'll be code/repo change either way, let's move this one through and do it as a follow-on.